### DISCUSSION TITLE
Does kubeedge support manual deployment of my application on the edge side if the network is down and cannot be restored in a short time?

### DISCUSSION THREAD

**Discussion by uonxhou** [kubeedge/kubeedge#5833] [Q&A]
In our scenario the network connection is very unstable and is likely to terminate for long periods of time. We need to manually deploy services to our devices by login to our side devices.

**Comment by magichan** [comment]
Within my knowledge scope, it should not be supported.

However, I believe it is a potential requirement for an edge case scenario. I suggest creating a kind/feature issue to see if the community is willing to implement this functionality.
